lib/gcloud/search/project.rb

```diff
@@ -17,6 +17,7 @@
 require "gcloud/search/connection"
 require "gcloud/search/credentials"
 require "gcloud/search/index"
+require "gcloud/search/index/list"
```

Comment:
As stated in the other require comment, I don't think the list require is needed here. It should be required with index.